apps/user-service/src/test/resources/application-test.yml

```
@@ -0,0 +1,14 @@
+    console:
+      enabled: true
+      path: /h2-console
+  datasource:
+    url: jdbc:h2:mem:testdb
+    driver-class-name: org.h2.Driver
+    username: sa
+    password: password
+  jpa:
+    database-platform: org.hibernate.dialect.H2Dialect
+    hibernate:
+      ddl-auto: update
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)